apps/src/javalab/JavalabView.jsx

```diff
@@ -72,10 +73,20 @@ class JavalabView extends React.Component {
 
   // Sends redux call to update dark mode, which handles user preferences
   renderSettings = () => {
-    const {isDarkMode, setIsDarkMode} = this.props;
+    const {displayTheme, setDisplayTheme} = this.props;
     return [
-      <a onClick={() => setIsDarkMode(!isDarkMode)} key="theme-setting">
-        Switch to {isDarkMode ? 'light mode' : 'dark mode'}
+      <a
+        onClick={() =>
+          setDisplayTheme(
+            displayTheme === DisplayTheme.DARK
+              ? DisplayTheme.LIGHT
+              : DisplayTheme.DARK
+          )
+        }
+        key="theme-setting"
+      >
+        Switch to{' '}
```

Comment:
(may be out of scope) can you make this translatable?

Reply:
Check out the most recent commit, there was no i18n in `JavalabView`, so translated a couple of strings. Hand tested:

![image](https://user-images.githubusercontent.com/25372625/148288106-f07f7d1d-88bf-4032-8b84-c9f97feadf41.png)